devel/addon/certmanager/install.sh

```diff
@@ -49,7 +49,7 @@ wait
 kubectl get namespace "${NAMESPACE}" || kubectl create namespace "${NAMESPACE}"
 
 # Install a copy of the CRDs
-kubectl apply -f "${REPO_ROOT}/deploy/charts/cert-manager/crds/"
+kubectl apply --validate=false -f "${REPO_ROOT}/deploy/manifests/00-crds.yaml"
```

Comment:
Hm, how come `--validate=false` is now required? 🤔

Reply:
was my fault... can be removed again I think